readme.md

```diff
@@ -0,0 +1,22 @@
+#Tunnetler#
+
+Is a revival of old-school computer game, called Tunneler (You can play this at: https://classicreload.com/tunneler.html). Major difference of Tunnetler in comparison to it's precceder will be network multiplayer. We dont want to bring Tunneler from the dead, we want to enhance it. So there are possible any reinforcements, which will make Tunnetler even more fun!
+
+##Technical info##
+
+Tunnetler is a desktop game created on Rust meetups in Bratislava's Progressbar. It is programmed exclusively in Rust language. We using Rust Piston game engine and various other Rust and Piston related crates.
+
+##Contribution##
+
+If you want to contribute, keep in mind following:
+
+- We are contributing using Pull Requests, which will be reviewed by Kixunil, and then approved or commented.
+- Any PR must be based upon existing Issue.
```

Comment:
I don't think this is necessary. I had good experience accepting PRs "out of blue". Especially various fixes.